docs/configuration.md

```diff
@@ -0,0 +1,16 @@
+# Configuration
```

Comment:
The same goes to this one, everything under `docs` that's not the `README` would look better having the first letter of the name uppercased.